### DISCUSSION TITLE
Quickfix for keyword case insensitive match

### DISCUSSION THREAD

**Discussion by rhewy** [microsoft/pylance-release#2624] [Enhancement]
Quikfix suggestion for 'undefined variable' when the var matches (case insensitive) a keyword

pylance 2020.11.0
Type checking set to strict but I would want this in all modes. 

See image below:
![image](https://user-images.githubusercontent.com/4993063/98450296-8c1eca00-20f0-11eb-98cf-c6bd3c278f80.png)

The user, probably meant True
Suggest to switch to True or whatever match is with a Keyword

**Comment by CatNofishing** [comment]
the same issue #67

**Comment by rhewy** [comment]
Definitely related ... need a fuzzy match

**Comment by rhewy** [comment]
This is what PostgreSQL does if I am off by a little bit:
![image](https://user-images.githubusercontent.com/4993063/98459031-78548180-214b-11eb-9ef0-8e644f5a2dab.png)

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
